cypress/integration/login.spec.ts

```diff
@@ -19,16 +19,33 @@ describe('Login process', () => {
         cy.get('[data-test="user-avatar"]').should('be.visible');
     });
 
-    it('clicking on user avatar takes you to My Page', () => {
+    it('takes you to My Page when clicking on user avatar', () => {
```

Comment:
When you make changes like this that are unrelated to the pull request, be sure to at least do it in a separate commit, if at all. When someone does `git blame` on a line of code, we want them to be able to understand from the commit message why the change was made. Doing so on this line now hints that the change was related to the logout button, which of course it's not.